### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by katakampranavi**
created a folder with sunbird-framework-management-ui-tool and also made suggested changes

## Project

Name of the project

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

**Comment by simran142002** [comment]
package lock and yarn has to be reverted

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
remove package-lock.json and yarn.lock

**Comment by katakampranavi** [comment]
@shubhigupta991 please check my commits and approve the PR

**Comment by shubhigupta991** [comment]
remove package.json file

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
remove the file package.json from git

**Comment by katakampranavi** [comment]
@shubhigupta991 removed package.json file as suggested

**Comment by katakampranavi** [comment]
@shubhigupta991 @simran142002 
can anyone please approve my pull request so that i can update my weekly status

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
You have not changed the slug line and haven't added your contact details in docs/2023/sunbird-framework-management-ui-tool/index.md file. Please make the following changes

**Comment by katakampranavi** [comment]
@shubhigupta991 made the changes as suggested. please check and confirm it.

**Comment by katakampranavi** [comment]
@shubhigupta991, I have made the changes as suggested. please check and confirm it.

**Comment by katakampranavi** [comment]
@shubhigupta991 @simran142002 can anyone please approve my pull request so that i can update my weekly status.

**Comment by katakampranavi** [comment]
@shubhigupta991 @simran142002 can anyone please approve my pull request so that i can update my weekly status.

**Comment by simran142002** [comment]
Your changes should be in 13 files you probably deleted package.json and yarn file instead of reverting them.

**Comment by katakampranavi** [comment]
@simran142002 as suggested by @shubhigupta991 i have removed the files kindly let me know what changes I need to do in current files

**Comment by katakampranavi** [comment]
@simran142002 I'm not getting it can you please elaborate what exactly i need to make changes to merge it in to branch apologies for inconvenience

File: docs/2023/sunbird-framework-management-ui-tool/updates/2023-07-14.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Milestones
+- [X] Requirement gathering and analysis
+- [X] UI Design and prototyping
+- [x] UI Development - Create New Framework  
+- [X] UI Development - List of Frameworks
+## Screenshots / Videos 
+
+
+## Contributions
+(https://github.com/Sunbird-Knowlg/sunbird-framework-management-ui-tool/tree/main/sunbird-framework-management-tool) this is the link after merging PR
```
Comment (shubhigupta991) [inline docs/2023/sunbird-framework-management-ui-tool/updates/2023-07-14.md:15]: Please attach the link of merged PR.

File: docs/2023/sunbird-framework-management-ui-tool/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+katakam pranavi
+
+
+# project name
+sunbird-framework-management-ui-tool
+
+
+# Contact Info
+[Email](mailto:pranavi123katakam@gmail.com)
+[Linked In](https://www.linkedin.com/in/pranavi-katakam35)
```
Comment (shubhigupta991) [inline docs/2023/sunbird-framework-management-ui-tool/index.md:18]: I have edited this link as this was incomplete. Please take a note for next time

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]
Can you elaborate the learnings for all the 3 weeks

**Comment by GeekGawd** [comment]
@katakampranavi Your folder structure is wrong, it should be like `organisation_name/project_name`

**Comment by GeekGawd** [comment]
@katakampranavi Have asked for some changes and also put folder inside Sunbird Knowldg right now its outside

**Comment by katakampranavi** [comment]
@GeekGawd i didnt get you .can you please elaborate it

**Review by shubhigupta991 — APPROVED** [review]

**Pull request merged**